src/components/Content/Content.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Wrapper from '../Wrapper';
+import ErrorBoundary from '../ErrorBoundary';
+import Filter from '../MovieFilter';
+import './style.css';
+
+const SearchResults = React.lazy(() => import('../SearchResults/SearchResults'));
+
+const Content = ({ searchResults, onFilterByGenre, onSorting }) => (
```

Comment:
why content?

Reply:
I mean the content where the movie cards are located. Should I rename?